config.go

```diff
@@ -252,6 +252,8 @@ type config struct {
 
 	RejectPush bool `long:"rejectpush" description:"If true, lnd will not accept channel opening requests with non-zero push amounts. This should prevent accidental pushes to merchant nodes."`
 
+	NoStaggerInitialReconnect bool `long:"no-stagger-initial-reconnect" description:"If false, will randomly stagger reconnections to persistent peers on startup between 0s and 30s. Nodes with a large number of channels should set this value to true to reduce large memory bursts when restarting."`
```

Comment:
Would prefer noting what happens when the flag is true instead. Also the second sentence doesn't seem to be relevant anymore, should perhaps mention this should be set when running on mobile.

Reply:
sgtm, fixed